README.md

```diff
@@ -21,6 +21,17 @@ use caesarlib::*;
 
 ## Methods
 ```rust
+
+// With custom base sequence
+// Can look like "ABC...Zabc...z"
+
+custom_encipher(base_seq: &str, offset: u16, message: &str) -> String
```

Comment:
Should we add a test to test that we can import it and actually change the sequence?